src/redux/initialState.js

```diff
@@ -38,6 +41,7 @@ const initialState = {
       id: 'aenean-ru-bristique-4',
       name: 'Aenean Ru Bristique 4',
       category: 'bed',
+      image: 'src=../../images/products/bed_4.jpg',
```

Comment:
sama nazwa zdjęcia, jeśli te dane będą przychodzić z BE to BE nie bedzie znał scieżki u Ciebie w aplikacji ani nie bedzie budował atrybutu `src`, tutaj powinno być jedynie `bed_4.jpg` a reszta powinna być zrobiona w komponencie